### DISCUSSION TITLE
Launch isympy in qtconsole

### DISCUSSION THREAD

**Discussion by avidseeker** [sympy/sympy#26364] [Show and tell]
I didn't see a documentation for this, and some (https://github.com/sympy/sympy/issues/17733, https://github.com/sympy/sympy/issues/5731) ask about it, but it is simply by running this on Qtconsole.

```python
import isympy; isympy.main();
```

and to pass on some option (e.g: `--auto`)
```python
import sys, isympy; isympy.main(); sys.argv.append('-a')
```

**Comment by asmeurer** [comment]
Do you make use of this? An `%isympy` magic would be more useful.

**Comment by cyan-ice** [comment]
It seems that you should append the option first, like:
```py
import sys, isympy; sys.argv.append('-I'); isympy.main()
```
Otherwise it just doesn't work for me.
